src/main/java/org/apache/sling/servlethelpers/MockSlingHttpServletRequest.java

```diff
@@ -516,6 +529,36 @@ public RequestParameterMap getRequestParameterMap() {
         return params;
     }
 
+    public void addRequestParameter(String name, String value) {
+    	if (this.parameterMap.containsKey(name)) {
```

Comment:
here (and in other lines) are tabs used instead of spaces.
please make sure to format your editor to use 4 spaces instead of tabs and reformat your changes.

Reply:
I've changed the settings in my IDE (Eclipse) and reformatted the whole file.  This has caused some incidental changes (putting spaces after commas, etc.) that I think are positive so I have left them in.  The only thing that might be an issue is that my line wrapping is at 180 chars.  A couple of lines that were previously split are now merged to one line.  Is that a problem?  I assume there is a style guide somewhere.  If you could point me to it, I will set my formatting settings appropriately.

Reply:
i think thats fine - just make sure to make a _separate_ commit for reformatting, so it's not mixed up with other changes.

coding standards: i think sling roughly follows https://felix.apache.org/documentation/development/coding-standards.html - but that may be a bit outdated.

i think the tabs -> space conversion is the important one, not important to reformat all other parts of the file.

Reply:
>  just make sure to make a separate commit for reformatting
Sorry, I already checked in commit that has the reformatting mixed.  I think it's OK however.  The changes weren't that extensive (tabs to spaces, longer lines in 2 cases, and single spaces inserted in a few places - such as after commas and between for and open parenthesis). 

Have a look at my latest commit.  If you are concerned I can back it out and separate out the reformatting.